src/index.js

```diff
@@ -0,0 +1,111 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const AWS = require('aws-sdk');
+
+const CONCLUSIONS = [
+	'neutral',
+	'skipped',
+	'success',
+	'cancelled',
+	'timed_out',
+	'action_required',
+	'failure',
+];
+
+// Fetch all the workflow run jobs
+const getJobs = async context => {
+	const RUN_ID = core.getInput('RUN_ID');
+	const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
```

Comment:
Do we really need to make GITHUB_TOKEN as input? Can we just use the current one to make the API calls?

Reply:
Completely removed as input as we're already passing the whole `github` object.